### ISSUE TITLE
Can we support Instant messaging

### ISSUE THREAD

**Issue by TechGuyVN** [datso/react-native-pjsip#103]
hi, 
Im using asterisk to setup pjsip,
How can we setup pjsip and our react-native source work with Instant messaging

Thanks you.

**Comment by rcorrie** [comment]
@TechGuyVN I don't think that has been fleshed out. It is definitely possible using pjsip, but the JS api is non-existent in this package. I might look into it when I get some free time.
